runtime/js/99_main.js

```diff
@@ -495,7 +495,24 @@ function bootstrapMainRuntime(runtimeOptions) {
   }
 
   if (unstableFlag) {
-    ObjectDefineProperties(globalThis, unstableWindowOrWorkerGlobalScope);
+    ObjectDefineProperties(
+      globalThis,
+      Object.values(unstableKeyedFeaturesForWindowOrWorkerGlobalScope)
+        .reduceRight((feature, o) => Object.assign(o, feature), {}),
+    );
+  } else {
+    for (
+      const unstableFeature of Object.keys(
```

Comment:
These will not pass lint checks - you'll need to use "primordials" here - `ObjectValues`, `ObjectAssign` and `ObjectKeys`. I would also suggest to do a for loop instead of `reduceRight`, but not big deal.

Reply:
Changed to use primordials.